scripts/upgrade/requestrr.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+. /etc/swizzin/sources/functions/requestrr
+
+if [[ ! -f /install/.requestrr.lock ]]; then
+    echo_error "Requestrr not detected. Exiting!"
+    exit 1
+fi
+
+_requestrr_download
+echo_progress_start "Replacing source code"
+
+unzip -q /tmp/requestrr.zip -d /tmp/ >> "$log" 2>&1
+rm /tmp/requestrr.zip
+cp -RT /tmp/requestrr*/ /opt/requestrr
+chown -R requestrr:requestrr /opt/requestrr
+rm -rf /tmp/requestrr*
+echo_progress_done "Extracted and overwrote existing files."
+
+echo_progress_start "Patching config"
+if [[ -f /install/.nginx.conf ]]; then
+    bash /usr/local/bin/swizzin/nginx/requestrr.sh
```

Comment:
this will not replace the nginx conf seeing as that script will only put it there it in case it does not exist, FYI

Reply:
lmao ``/install/.nginx.conf`` never exists... i must've written this when I was dumb.